### DISCUSSION TITLE
AtomicEnergies input for training new model

### DISCUSSION THREAD

**Discussion by lamthuy** [ACEsuit/mace#402] [Q&A] [answered]
Hi,
I see that in all pretrained models it use AtomicEnergyBlock with initial energy filled for each atom:
```
AtomicEnergiesBlock(energies=[-3.6672, -1.3321, -3.4821, -4.7367, -7.7249, -8.4056, -7.3601, -7.2846, -4.8965, 0.0000, -2.7594, -2.8140, -4.8469, -7.6948, -6.9633, -4.6726, -2.8117, -0.0626, -2.6176, -5.3905, -7.8858, -10.2684, -8.6651, -9.2331, -8.3050, -7.0490, -5.5774, -5.1727, -3.2521, -1.2902, -3.5271, -4.7085, -3.9765, -3.8862, -2.5185, 6.7669, -2.5635, -4.9380, -10.1498, -11.8469, -12.1389, -8.7917, -8.7869, -7.7809, -6.8500, -4.8910, -2.0634, -0.6396, -2.7887, -3.8186, -3.5871, -2.8804, -1.6356, 9.8467, -2.7653, -4.9910, -8.9337, -8.7356, -8.0190, -8.2515, -7.5917, -8.1697, -13.5927, -18.5175, -7.6474, -8.1230, -7.6078, -6.8503, -7.8269, -3.5848, -7.4554, -12.7963, -14.1081, -9.3549, -11.3875, -9.6219, -7.3244, -5.3047, -2.3801, 0.2495, -2.3240, -3.7300, -3.4388, -5.0629, -11.0246, -12.2656, -13.8556, -14.9331, -15.2828])
```
And that is also the required input if we want to train the model for other data. I am not sure from where you get this energy?

**Comment by ilyes319** [comment, marked as answer]
This corresponds to the energy that MACE will predicted for an isolated atom. There are two ways one can set them in MACE:
- To get the exact limit of dissociating atoms, you will need to compute in your electronic structure code the energy of an atom isolated.
- If you do not care about bound breaking, you can set --E0s="average".

You can read the section B. Data normalization page 17 of this paper to know more : https://arxiv.org/pdf/2205.06643.
